### DISCUSSION TITLE
Vagrant box management (add, list and remove)

### DISCUSSION THREAD

**Discussion by Rockin2** [saltstack/salt#67279] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
We want to add vagrant boxes with Salt. (vagrant box add)

**Describe the solution you'd like**
A function in the vagrant modules to install a vagrant box. Two others function to remove and list boxes could be good also.

**Describe alternatives you've considered**
Using cmd.run function. It works but it is not really clean.

**Comment by s0undt3ch** [comment]
Thank you for the feature submission @Rockin2
